draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -1110,8 +1110,8 @@ ABORT if:
 
 Procedure:
 
-1. c_arr = (Abar, Bbar, D, T1, T2, R, i1, ..., iR,
-                                            msg_i1, ..., msg_iR, domain)
+1. c_arr = (R, i1, msg_i1, ..., iR, msg_iR, domain,
```

Comment:
1. c_arr = (R, i1, ..., iR, msg_i1, ..., msg_iR, domain,

Reply:
My mistake. Didn't understand the syntax.

Reply:
Added an extended description of the inputs to the challenge digest calculation. Hopefully this should better explain the intended ordering of the index, message pair in `c_arr`.